### ISSUE TITLE
pdb4amber?

### ISSUE THREAD

**Issue by jchodera** [choderalab/ambermini#29]
Does `pdb4amber` get installed with ambermini?  I don't see it getting put into the path.

**Comment by swails** [comment]
No, it's not.  But it would be easy to add -- it's a simple Python program with no external dependencies.

**Comment by swails** [comment]
It's also currently Py2-only, to my knowledge (I don't know if `2to3` will work on it -- I haven't tried).

**Comment by juliebehr** [comment]
I agree that this would be helpful to have in ambermini!
